xds/src/main/java/io/grpc/xds/Bootstrapper.java

```diff
@@ -37,87 +37,51 @@
 import javax.annotation.concurrent.Immutable;
 
 /**
- * Loads configuration information to bootstrap xDS load balancer.
+ * Loads configuration information to bootstrap gRPC's integration of xDS protocol.
  */
-@Immutable
 abstract class Bootstrapper {
```

Comment:
`@NotThreadSafe`

Reply:
Actually it should be `@ThreadSafe` (although I don't think we need to add such as annotation).

Reply:
It depends on your implementation.